### PR TITLE
Home redesign

### PR DESCRIPTION
**Pull request by DmytroDidukh**

### CODE REVIEW

File: src/containers/search-bar-list/search-bar-list-item/search-bar-list-item.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+          <div>
+            {product.basePrice[currency].value / 100}{' '}
+            {product.basePrice[currency].currency}
+          </div>
+        </div>
+        <div className={styles.buttons}>
+          <ClassicButton
+            buttonType='button'
+            innerText={<ShoppingBasketIcon />}
+            onClickHandler={() => console.log('to cart')}
```
Comment (moran711) [inline src/containers/search-bar-list/search-bar-list-item/search-bar-list-item.js:50]: Please remove this console.log()

File: src/containers/search-bar-list/search-bar-list-item/search-bar-list-item.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            buttonStyle='classic'
+          />
+          <ClassicButton
+            buttonType='button'
+            innerText='ДЕТАЛІ'
```
Comment (Iryna-Bzdyr) [inline src/containers/search-bar-list/search-bar-list-item/search-bar-list-item.js:55]: Try to replace this hardcode by adding it to the config file

File: src/containers/search-bar-list/search-bar-list.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    products: SearchBar.list,
+    searchBarVisibility: SearchBar.visibility,
+    searchBarLoading: SearchBar.loading,
+    language: Language.language
+  }));
+
+  const styles = useStyles();
+  const dispatch = useDispatch();
+
+  useEffect(() => () => dispatch(setProductsForSearchBar([])), [dispatch]);
```
Comment (Iryna-Bzdyr) [inline src/containers/search-bar-list/search-bar-list.js:27]: Why you use this hook?

File: src/pages/home/contructor-preview/constructor-preview.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    >
+      <ReactPlayer
+        width='100%'
+        height='100%'
+        playing={isPlaying}
+        volume={0}
+        playsinline={false}
+        loop
+        muted
+        url='https://www.youtube.com/watch?v=KG1y-lk2f0E&ab_channel=Ukra%D1%97ner'
```
Comment (Iryna-Bzdyr) [inline src/pages/home/contructor-preview/constructor-preview.js:42]: Try to replace this hardcode

File: src/pages/home/contructor-preview/constructor-preview.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+      />
+      <div
+        className={styles.constructorInner}
+        onMouseLeave={() => setIsMouseIn(false)}
+        onMouseEnter={() => setIsMouseIn(true)}
+      >
+        <ExpandLessIcon className={styles.constructorInnerIcon} />
+        <Link to='/constructor' className={styles.constructorInnerLink}>
+          {HOME_BUTTONS[language].MOVE_TO_CONSTRUCTOR}
+          <span>&#8594;</span>
```
Comment (Iryna-Bzdyr) [inline src/pages/home/contructor-preview/constructor-preview.js:52]: Try to replace this hardcode unicode with a materialUI icon or add it to the config file

File: src/pages/home/modles-list/model-item/model-item.js [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+      <div className={styles.modelItemTitle}>{model.name[language].value}</div>
+      <div className={styles.modelItemImage}>
+        <img src={IMG_URL + model.images.small} alt='model' />
+      </div>
+      <Link
+        className={styles.link}
+        to={`/${model.category.name[1].value.toLowerCase()}/${model.name[1].value.toLowerCase()}`}
+      >
+        {HOME_BUTTONS[language].MOVE_TO_MODEL}
+        <span>&#8594;</span>
```
Comment (Iryna-Bzdyr) [inline src/pages/home/modles-list/model-item/model-item.js:55]: Try to replace this hardcode unicode with a materialUI icon or add it to the config file

File: src/pages/home/modles-list/models-list.js [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+      <div className={styles.modelsWrapper}>
+        {models.map((model) => (
+          <ModelItem key={model._id} model={model} />
+        ))}
+        <Link className={modelItemStyles.modelItem} to='/unique'>
+          <div className={modelItemStyles.modelItemTitle}>Unique</div>
+          <div
+            className={modelItemStyles.modelItemImage}
+            style={{
+              background: `url(https://i.imgur.com/ygJaph4.jpg) center center`,
```
Comment (Iryna-Bzdyr) [inline src/pages/home/modles-list/models-list.js:49]: Try to replace this hardcode

File: src/containers/search-bar-list/search-bar-list.js (hunk 3 above)
Comment (DmytroDidukh) [reply src/containers/search-bar-list/search-bar-list.js:27]: It was necessary for clearing state when `Search Bar List `component are going to `unmount`. But you are right, no need to clear list.

**Review by Iryna-Bzdyr — APPROVED** [review]

**Review by moran711 — APPROVED** [review]

**Review by nadiyafomenko — APPROVED** [review]

**Pull request merged**